Assets/Scripts/Scenes/InGame/Player/Item/ItemData.cs

```diff
@@ -0,0 +1,22 @@
+using Scenes.Ingame.Player;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "ItemData", menuName = "ScriptableObjects/ItemData")]
+public class ItemData : ScriptableObject
+{
+    [Header("�A�C�e���̌ŗL�l")]
+    public int itemID;
+    public string itemName;
+    public Sprite itemSprite;
+
+    [Header("�V���b�v�֘A�̏��")]
+    public int price;
+    public string itemInfo;
+    public ItemType type;
+
+    [Header("�C���Q�[���̐ݒ�")]
+    [Tooltip("�g���؂�̃A�C�e���Ȃ�true��")]
+    public bool isSingleUse;
+    public ItemEffect thisItemEffect;//inspecter��ł͋�̂܂܂ő��v
```

Comment:
```suggestion
[HideInInspector]
 public ItemEffect thisItemEffect;
``` 
[HideInInspector]の属性をつけることでInspecter上に表示しないようにできます

Reply:
変更しました。情報ありがとうございます！